src/player.py

```diff
@@ -3,23 +3,34 @@
 class Player(Game):
     # initialize hand with games deck
     def __init__(self, game):
+        print("I compile")
```

Comment:
This constructor should not take game as a parameter. Why would it need to if games deck is public